Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -1,135 +1,158 @@
-                  <Dropdown.Header>
-                    <div className="flex items-center gap-3 mb-2">
-                      <Avatar
-                        alt="User settings"
-                        img={user.photoURL || "https://flowbite.com/docs/images/people/profile-picture-5.jpg"}
-                        rounded
-                        size="sm"
+                <div className="relative">
+                  <button 
+                    onClick={() => setShowUserMenu(!showUserMenu)}
+                    className="flex items-center focus:outline-none"
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)